db/migrate/20200428205535_uniqueness_rule_references_rule_index.rb

```diff
@@ -0,0 +1,28 @@
+class UniquenessRuleReferencesRuleIndex < ActiveRecord::Migration[5.2]
+  class NewRuleReferencesRule < ApplicationRecord; end
+
+  def change
+    ids_to_keep = RuleReferencesRule.group(
```

Comment:
```suggestion
    ids_to_keep = RuleReferencesRule.select(:rule_id, :rule_reference_id).group(
```

Reply:
This one doesn't make sense I think - I only want the IDs here

Reply:
Ok, then `select(:id)` - the idea is that limiting your select instead of the default `select('*')` makes the response quicker.